pod.go

```diff
@@ -113,12 +113,15 @@ type Volumes []Volume
 // Set assigns volume values from string to a Volume.
 func (v *Volumes) Set(volStr string) error {
 	volSlice := strings.Split(volStr, " ")
+	const expectedVolLen = 2
+	const volDelimiter = ":"
 
 	for _, vol := range volSlice {
-		volArgs := strings.Split(vol, ":")
+		volArgs := strings.Split(vol, volDelimiter)
 
-		if len(volArgs) != 2 {
-			return fmt.Errorf("Wrong string format: %s, expecting only 2 parameters separated with ':'", vol)
+		if len(volArgs) != expectedVolLen {
+			return fmt.Errorf("Wrong string format: %s, expecting only %v parameters separated with %q",
```

Comment:
I understand the sentiment, but you do only need to look up 3 lines to see the definition ;) But if we don't use a variable, that error will bitrot - duplication is evil.